modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ExecutionContext.java

```diff
@@ -41,16 +42,14 @@
 import org.apache.ignite.internal.sql.engine.exec.mapping.ColocationGroup;
 import org.apache.ignite.internal.sql.engine.exec.mapping.FragmentDescription;
 import org.apache.ignite.internal.sql.engine.type.IgniteTypeFactory;
-import org.apache.ignite.internal.sql.engine.util.AbstractQueryContext;
-import org.apache.ignite.internal.sql.engine.util.BaseQueryContext;
 import org.apache.ignite.internal.sql.engine.util.TypeUtils;
 import org.apache.ignite.network.ClusterNode;
 import org.jetbrains.annotations.Nullable;
 
 /**
  * Runtime context allowing access to the tables in a database.
  */
-public class ExecutionContext<RowT> extends AbstractQueryContext implements DataContext {
```

Comment:
seems AbstractQueryContext no need anymore and can be merged with a BaseQueryContext

Reply:
also interface Context now looks strange ) maybe will be better to rename it

Reply:
> seems AbstractQueryContext no need anymore

done

> also interface Context now looks strange

not sure which one... Elaborate please